.github/workflows/build-mac.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
       - name: Install PETSc
         run: |
-          git clone --depth 1 https://github.com/firedrakeproject/petsc.git
+          git clone --depth 1 --branch v3.22.2 https://gitlab.com/petsc/petsc.git
```

Comment:
FYI, 3.22.4 was released a couple of weeks ago

Reply:
I know. Our current fork is on 3.22.2 so I want to do the switch in two stages: first switch to upstream and then update to the latest release.

Then we will need to start thinking about which versions of PETSc Firedrake `main` (we will rename `master`) and `release` point to when we begin doing releases.